client/pages/background.jsx

```diff
@@ -30,7 +30,7 @@ export default class Background extends React.Component {
     return (
       <div className="container-fluid background-fixed">
         <div className="btn-group-vertical gap-3 mt-3">
-          <a href={''}>
+          <a href={'#addJokes'}>
```

Comment:
If your attribute value is just a string you don't need to use curly brackets, you can just use quotes:

```JSX
<a href="#addJokes">
```